samples/c/text_generation/CMakeLists.txt

```diff
@@ -12,9 +12,8 @@ function(add_sample_executable target_name)
     add_executable(${target_name} ${target_name}.c)
     # Specifies that the source file should be compiled as a C source file
     set_source_files_properties(${target_name}.c PROPERTIES LANGUAGE C)
-    target_link_libraries(${target_name} PRIVATE openvino::genai)
+    target_link_libraries(${target_name} PRIVATE openvino::genai::c)
     set_target_properties(${target_name} PROPERTIES
-        COMPILE_PDB_NAME ${target_name}
```

Comment:
I see COMPILE_PDB_NAME removed here, may I know if PDB symbol in Release build will be enabled later with openvino [PR29402](https://github.com/openvinotoolkit/openvino/pull/29402) and propagate to openvino.genai?

Reply:
this property is used incorrectly in both OpenVINO and GenAI.

To have PDB files in Release, you need to compile both OpenVINO and GenAI with that option from PR you mentioned.

Reply:
Got it, thanks!